Jenkinsfile

```diff
@@ -153,15 +158,26 @@ void checkoutQuarkusRepo() {
     }
 }
 
-MavenCommand getMavenCommand(String directory, boolean addQuarkusVersion=true) {
+MavenCommand getMavenCommand(String directory, boolean addQuarkusVersion=true, boolean canNative = false) {
     mvnCmd = new MavenCommand(this, ['-fae'])
                 .inDirectory(directory)
     if (addQuarkusVersion && getQuarkusBranch()) {
         mvnCmd.withProperty('version.io.quarkus', '999-SNAPSHOT')
     }
+    if(canNative && isNative()) {
+        mvnCmd.withProfiles(['native'])
```

Comment:
I think we should also add this property:
`-Dquarkus.profile=native`
The test failure in the PR check is probably caused by https://github.com/quarkusio/quarkus/issues/13341.
We recommend setting the property in the readme for running the native compilation.

I can see the issue has been closed, so hopefully with the Quarkus 1.12.0.Final it will work without this workaround.

Reply:
added with comment